src/app/modals/LoginModal/LoginModal.tsx

```diff
@@ -27,21 +25,20 @@ const LoginModal: React.FC<LoginModalProps> = ({ isOpen, onClose }) => {
     password: Yup.string().required("Password is required"),
   });
 
-  const handleSubmit = async (values: { username: string; password: string }) => {
+  const handleSubmit = async (
+    values: { username: string; password: string }, 
+    actions: any
+  ) => {
+    const {setSubmitting, setStatus } = actions;
     try {
-      setError(null); 
-      await login(values.username, values.password);
+      await login(values.username, values.password); 
       onClose();
     } catch (err) {
       console.error("Login failed:", err);
-      setError("Login failed. Please try again."); 
+      setStatus("Login failed. Please try again.");
+      setSubmitting(false);
```

Comment:
`setSubmitting` ne için kullanıldı?

Reply:
abi formigin bi ozelligi form gonderimini kontol ediyor